test/functional/apps/visualize/_input_control_vis.js

```diff
@@ -39,218 +39,206 @@ export default function ({ getService, getPageObjects }) {
-          await PageObjects.visualize.clearComboBox('listControlSelect0');
-          await PageObjects.visualize.setComboBox('listControlSelect0', 'osx');
-          await testSubjects.click('inputControlSubmitBtn');
-
-          const hasOldFilter = await filterBar.hasFilter(FIELD_NAME, 'ios');
-          const hasNewFilter = await filterBar.hasFilter(FIELD_NAME, 'osx');
-          expect(hasOldFilter).to.equal(false);
-          expect(hasNewFilter).to.equal(true);
-        });
-
-        it('should clear dropdown when filter pill removed', async () => {
-          await filterBar.removeFilter(FIELD_NAME);
-          await PageObjects.common.sleep(500); // give time for filter to be removed and event handlers to fire
-
-          const hasValue = await PageObjects.visualize.doesComboBoxHaveSelectedOptions('listControlSelect0');
-          expect(hasValue).to.equal(false);
-        });
-
-        it('should clear form when Clear button is clicked but not remove filter pill', async () => {
-          await PageObjects.visualize.setComboBox('listControlSelect0', 'ios');
-          await testSubjects.click('inputControlSubmitBtn');
-          const hasFilterBeforeClearBtnClicked = await filterBar.hasFilter(FIELD_NAME, 'ios');
-          expect(hasFilterBeforeClearBtnClicked).to.equal(true);
-
-          await testSubjects.click('inputControlClearBtn');
-          const hasValue = await PageObjects.visualize.doesComboBoxHaveSelectedOptions('listControlSelect0');
-          expect(hasValue).to.equal(false);
-
-          const hasFilterAfterClearBtnClicked = await filterBar.hasFilter(FIELD_NAME, 'ios');
-          expect(hasFilterAfterClearBtnClicked).to.equal(true);
-        });
-
-        it('should remove filter pill when cleared form is submitted', async () => {
-          await testSubjects.click('inputControlSubmitBtn');
-          const hasFilter = await filterBar.hasFilter(FIELD_NAME, 'ios');
-          expect(hasFilter).to.equal(false);
-        });
+      it('should not display staging control buttons', async () => {
+        const submitButtonExists = await testSubjects.exists('inputControlSubmitBtn');
+        const cancelButtonExists = await testSubjects.exists('inputControlCancelBtn');
+        const clearButtonExists = await testSubjects.exists('inputControlClearBtn');
+        expect(submitButtonExists).to.equal(false);
+        expect(cancelButtonExists).to.equal(false);
+        expect(clearButtonExists).to.equal(false);
+      });
+
+      it('should add filter pill when item selected', async () => {
+        await PageObjects.visualize.setComboBox('listControlSelect0', 'ios');
+
+        const selectedOptions = await PageObjects.visualize.getComboBoxSelectedOptions('listControlSelect0');
+        expect(selectedOptions[0].trim()).to.equal('ios');
+
+        const hasFilter = await filterBar.hasFilter(FIELD_NAME, 'ios');
+        expect(hasFilter).to.equal(true);
```

Comment:
👌 I think this is more formatting, I could not tell if there were any functional changes, but appears maybe just some page loading methods were removed.   I did run the new and old files and noticed the results were the same.